neco-server-exporter/pkg/collector/bpf/collector.go

```diff
@@ -0,0 +1,135 @@
+package bpf
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/fs"
+	"strconv"
+
+	"github.com/cilium/cilium/pkg/client"
+	"github.com/cilium/ebpf"
+	"github.com/cybozu/neco-containers/neco-server-exporter/pkg/exporter"
```

Comment:
```suggestion
	"github.com/cilium/cilium/pkg/client"
	"github.com/cilium/ebpf"

	"github.com/cybozu/neco-containers/neco-server-exporter/pkg/exporter"
```

Missing blank line between external and internal imports.

Reply:
I didn't expect to have this comment, because the proposed style is not currently adopted by our team.

- Codes dividing external and internal imports
  - https://github.com/cybozu/neco-containers/blob/main/cep-checker/main.go
  - https://github.com/cybozu-go/tenet/blob/main/cmd/tenet-controller/main.go
- Codes not doing so
  - https://github.com/cybozu/neco-containers/blob/main/admission/cmd/run.go
  - https://github.com/cybozu-go/coil/blob/main/v2/cmd/coil-installer/sub/root.go

I'm going to install `goimports` in `tools.mk` and check the style in `make check-generate`.